controller/transfer/manager.py

```diff
@@ -39,6 +39,12 @@
 from datetime import datetime
 from util import notification
 from sqlalchemy.sql import text as sql_text
+from controller.labeling_task import manager as labeling_task_manager
+from controller.labeling_task_label import manager as labeling_task_label_manager
+from submodules.model.business_objects.record_label_association import (
+    count_null_labels,
+    update_null_labels,
+)
```

Comment:
we shouldn't import functions directly form the submodules to preserve the context in the code.
```suggestion
from submodules.model.business_objects import record_label_association as rla
```